common/src/main/java/me/lucko/luckperms/common/calculator/LuckyPerms.java

```diff
@@ -0,0 +1,145 @@
+ *  of this software and associated documentation files (the "Software"), to deal
+ *  in the Software without restriction, including without limitation the rights
+ *  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ *  copies of the Software, and to permit persons to whom the Software is
+ *  furnished to do so, subject to the following conditions:
+ *
+ *  The above copyright notice and this permission notice shall be included in all
+ *  copies or substantial portions of the Software.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ *  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ *  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ *  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ *  SOFTWARE.
+ */
+
+package me.lucko.luckperms.common.calculator;
+
+import net.luckperms.api.util.Tristate;
+
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Locale;
+import java.util.Set;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * The LuckPerms Luckiness Calculator (tm)
+ */
+public final class LuckyPerms {
+
+    /** A sequence of the first 1000 lucky numbers. */
+    private static final Set<Integer> LUCKY_NUMBERS = generateLuckyNumbers(1000);
+    /** Is it an unlucky day? */
+    private static final boolean UNLUCKY_DAY;
+    /** The unlucky number. */
+    private static final int UNLUCKY_NUMBER = 666;
+    /** The lucky number */
+    private static final int LUCKY_NUMBER;
+
+    static {
+        LocalDate date = LocalDate.now();
+        UNLUCKY_DAY = date.getDayOfWeek() == DayOfWeek.FRIDAY && date.getDayOfMonth() == 13;
+
+        if (Locale.getDefault().getLanguage().equals(Locale.CHINESE.getLanguage())) {
+            // https://en.wikipedia.org/wiki/Chinese_numerology#Eight
+            LUCKY_NUMBER = 8;
+        } else {
+            // https://en.wikipedia.org/wiki/7
+            LUCKY_NUMBER = 7;
+        }
+    }
+
+    /** A ~magical~ counter used when determining luckiness. */
+    private final AtomicInteger counter;
+
+    public LuckyPerms() {
+        this.counter = new AtomicInteger();
+    }
+
+    /**
+     * Uses highly advanced, patented LuckPerms(tm) calculations to determine
+     * the current luckiness measure.
+     *
+     * <p>Returns {@link Tristate#TRUE} if lucky, {@link Tristate#FALSE} if unlucky,
+     * and {@link Tristate#UNDEFINED} if unable to determine.</p>
+     *
+     * @return the calculated luckiness measure
+     */
+    public Tristate calculateLuckiness() {
+        int i = this.counter.incrementAndGet();
```

Comment:
Greetings,

I hate to have to even bring this up, but it seems to appear that your variable is named very similarly to a variable I used previously in one of my projects ([terrible-plugin](https://github.com/Jaimss/terrible-plugin/blob/a72870845a89cae6903a87882db1e6ae271be376/src/main/java/dev/jaims/terribleplugin/Main.java#L46)). I would really appreciate your renaming this variable to something else. Currently this looks like thievery, which I am going to assume, for the benefit of both parties, was not your intention. I figured it would be nicest to speak to you about this before taking legal action. 

You have 12 seconds.

Regards,
James Harrell